tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -812,8 +812,171 @@ def test_removing_label_sparse_feature_sizes(
     final_sparse_feature_sizes: Dict[Text, Dict[Text, List[int]]],
     label_attribute: Text,
 ):
+    """Tests if label attribute is removed from sparse feature sizes collection."""
     sparse_feature_sizes = DIETClassifier._remove_label_sparse_feature_sizes(
         sparse_feature_sizes=initial_sparse_feature_sizes,
         label_attribute=label_attribute,
     )
     assert sparse_feature_sizes == final_sparse_feature_sizes
+
+
+@pytest.mark.timeout(120)
+async def test_adjusting_layers_incremental_training(
+    component_builder: ComponentBuilder, tmpdir: Path
+):
+    """Tests adjusting sparse layers of `DIETClassifier` to increased sparse
+       feature sizes during incremental training.
+
+       Testing is done by checking the layer sizes.
+       Checking if they were replaced correctly is also important
+       and is done in `test_replace_dense_for_sparse_layers`
+       in `test_rasa_layers.py`.
+       """
```

Comment:
Align the docstring quotes with what's at the starting.
```suggestion
 """
```